babel.npm.cjs.js

```diff
@@ -8,8 +8,16 @@ const presets = [
   [
     '@babel/preset-env',
     {
+      modules: 'commonjs',
       targets: {
-        node: true
+        browsers: [
```

Comment:
Is there a shared file that all these webpack files could reference for these babel targets so that we only have to maintain one file?  Theres starting to be a lot of these duplicates

Reply:
I believe we have, or need to get, a ticket in to cleanup our build scripts and processes.

Reply:
Ticket for this: https://issues.newrelic.com/browse/NR-102558